content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-core.md

```diff
@@ -360,6 +360,8 @@ export DD_RUNTIME_METRICS_ENABLED=true
 dotnet example.dll
 ```
 
+<div class="alert alert-info"> If you are using Alpine Linux, set the <code>CORECLR_PROFILER_PATH</code> environment variable to a path for musl based distributions: <code>linux-musl-x64/</code>.</div>
```

Comment:
I think this note is going to cause confusion. We have one Linux package for glibc distributions and one Linux package for musl distributions, and both of them install a loader native binary to `/opt/datadog/Datadog.Trace.ClrProfiler.Native.so` so that configuration is consistent. This removes the need to specify a `linux-musl-x64` or `linux-x64` path that is platform-dependent